spec/unit/mupdf_spec.lua

```diff
@@ -137,6 +139,29 @@ describe("mupdf module", function()
                 md5.sumFile(simple_pdf_compare)
             )
         end)
+        it("should open a page, add contents to an existing annotation and write a new document", function()
+            local ffi = require("ffi")
+            local doc = M.openDocument(simple_pdf_out)
+            assert.is_not_nil(doc)
+            local page = doc:openPage(1)
+            assert.is_not_nil(page)
+            local annot = page:getMarkupAnnotation(ffi.new("float[8]", {
+                 70,  930,
+                510,  930,
+                510,  970,
+                 70,  970 }),
+                1)
+            page:updateMarkupAnnotation(annot, "annotation contents")
+            page:close()
+            doc:writeDocument(simple_pdf_out_annotated)
+            local out_f = io.open(simple_pdf_out_annotated, "r")
+            out_f:read("*a")
+            out_f:close()
```

Comment:
Also applies to the existing test above: I have no idea what this is attempting to test :D.

Reply:
Thanks, removed it in both tests.